tests/scalers_go/gcp_stackdriver/gcp_stackdriver_test.go

```diff
@@ -111,9 +130,10 @@ spec:
     - type: gcp-stackdriver
       metadata:
         projectId: {{.ProjectID}}
-        filter: 'metric.type="storage.googleapis.com/network/received_bytes_count" AND resource.type="gcs_bucket" AND metric.label.method="WriteObject" AND resource.label.bucket_name="{{.BucketName}}"'
-        metricName: {{.BucketName}}
+        filter: 'metric.type="pubsub.googleapis.com/topic/num_unacked_messages_by_region" AND resource.type="pubsub_topic" AND resource.label.topic_id="{{.TopicName}}"'
+        metricName: {{.TopicName}}
         targetValue: "5"
+		activationTargetValue: "{{.ActivationThreshold}}"
```

Comment:
This will not work, the template indentation needs to be done with spaces, not tabs.
Maybe I'm wrong, e2e test will validate it xD